grblas/vector.py

```diff
@@ -233,6 +232,9 @@ def ewise_add(self, other, op=None, *, require_monoid=True):
         if op is None:
             op = monoid.plus
         opclass = find_opclass(op)
+        if opclass.startswith('Parameterized'):
```

Comment:
This code is duplicated in almost every function. Can we bury it inside `find_opclass` somehow?

Reply:
Done.  Also added tests for parameterized BinaryOp as accum.